Mathlib/Data/Finsupp/Defs.lean

```diff
@@ -140,6 +140,8 @@ theorem ext_iff {f g : α →₀ M} : f = g ↔ ∀ a, f a = g a :=
   FunLike.ext_iff
 #align finsupp.ext_iff Finsupp.ext_iff
 
+lemma ne_iff : f ≠ g ↔ ∃ a, f a ≠ g a := FunLike.ne_iff
```

Comment:
Let's not introduce a `variable` for just one lemma, since it pollutes the goal state in all the other lemmas.
```suggestion
lemma ne_iff {f g : α →₀ M} : f ≠ g ↔ ∃ a, f a ≠ g a := FunLike.ne_iff
```